docs/examples/promise/03_listen_to_balance_change/index.js

```diff
@@ -25,10 +25,8 @@ async function main () {
     if (change.isZero()) {
       return;
     }
-
     previous = current;
-
-    console.log(`Balance of ${Alice}: ${current}, ${change} change`);
+    console.log(`New transaction of: ${change}`);
```

Comment:
balance of

Reply:
As in: `New transaction balance of: ${change}` ? If yes, then I changed it in both versions.

Reply:
Not sure why we are using the work "transaction" here - balances can change without receiving anything. "account balance" does make sense, just "transaction" or "transaction + balance" does not quite.

Reply:
I was just trying to find a better wording because the log was looking a bit cluttered/ not easy to grasp with the hash, the balance (1152921504606686330) and the change. 
But I have no strong feelings about that, I can change it back.